sdk/tables/azure-data-tables/samples/creation_deletion_of_table.py

```diff
@@ -17,7 +17,7 @@ def connection_string_auth(self):
 
     def sas_token_auth(self):
         from azure.data.tables import TableServiceClient
-        from azure.data.tables._shared.table_shared_access_signature import generate_account_sas
+        from azure.data.tables.table_shared_access_signature import generate_account_sas
```

Comment:
should just be able to add this import on the previous line since you can import `generate_account_sas` directly from `azure.data.tables` (it's exposed in the init file at the azure/data/tables level)

Reply:
As an aside: do these samples still work? If you haven't yet can you try running all samples and make sure they pass with all of these newe changes?

Reply:
These samples do not work, creating a separate issue for re-building these